tensorflow-notebook-gpu/test/test_tensorflow.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Jupyter Development Team.
```

Comment:
While I agree it would be good to test GPU stuff here, I'm not sure it will be possible to test functional stuff. For example, what I noticed when I created a non-functional GPU docker image is that `nvidia-smi` ran without a problem and I could see the GPU info, however, for CUDA version it would say `N/A` and as such the image was not usable. Doing a test like that here, or something like `torch.cuda.is_available()` in pytorch would be great, but it would require a GPU on the machine running the tests. 
Do you have an idea of what type of tests could be done without requiring a GPU?

Reply:
That's tricky, yes.
To be honest, I'm not familiar with GPUs at all, so no ideas so far.

Reply:
For now, I just say that we run nvidia-smi, as you did and check that error code is zero.
That's not perfect, but better than nothing.